hack/tool/backup_restore/README.md

```diff
@@ -4,6 +4,20 @@ This go module is a command line tool to back up and restore the configuration a
 
 It has two subcommands `backup` and `restore`.
 
+## `backup`
+
+`backup` can be used to back up the Configuration object managed by terraform-controller and the Terraform state (if the configuration uses the Terraform Kubernetes backend).
+
+The main usage of the `backup` subcommand is:
+
+```shell
+go main.go backup --name <name of the Configuration> --namespace <namespace of the Configuration>
+```
+
+Then you will get the `cofiguration.yaml` and the `state.json` in the workdir.
+
+Next, you can restore the Configuration and the Terraform state in another Kubernetes cluster using the `restore` subcommand.
```

Comment:
The configuration could not get restored as some fields of metadata need to be cleaned before restoring.

Reply:
Actually, the configuration will be cleaned before encode it into yaml.

You can checkout the `cleanUpConfiguration` function [here](https://github.com/kubevela/terraform-controller/pull/315/files#diff-ef92b8134a03fc52abb41ba26a23883a872368e9cfb5d993395a328e3f0c9a8eR65).